src/main/java/scripts/versionscripts/ReadFromTerraHelmFileRepoAndGitCommitLog.java

```diff
@@ -0,0 +1,49 @@
+package scripts.versionscripts;
+
+import bio.terra.testrunner.runner.VersionScript;
+import bio.terra.testrunner.runner.VersionScriptResult;
+import bio.terra.testrunner.runner.config.ServerSpecification;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReadFromTerraHelmFileRepoAndGitCommitLog extends VersionScript {
```

Comment:
Instead of combining the two version lookups in a single script, can you take advantage of your change to allow multiple version scripts per server? i.e. use both `ReadFromGitCommitLog` and `ReadFromTerraHelmfileRepo`

Reply:
Multiple version script already supported in latest `ServerSpecification`.

Reply:
Do you still need this class?

Reply:
Removed `ReadFromTerraHelmFileRepoAndGitCommitLog` class.